### ISSUE TITLE
Reg\Operations - Internal User can see Operation grid

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1705]
#### Description:
As an internal user, I can see all of the operations that have been registered

#### Acceptance Criteria:
Given I am on the [BCIERS dashboard](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7781-6471&t=EdbpdHgRnvGxWEpV-4) 
When I click Operations
Then I am brought to the operations grid
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7861-688&t=EdbpdHgRnvGxWEpV-4)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by andrea-williams** [comment]
@zoeyli-46  could you please update the Figma link? It's currently linked to an Archived frame

**Comment by zoeyli-46** [comment]
its updated

**Comment by patrickisaac** [comment]
Based on discussion with the team, this has already been implemented. Closing this ticket.
